site/src/utils/utils.js

```diff
@@ -25,3 +25,30 @@ export async function timed(fn, label, ctx) {
     sentry.captureException(err)
   }
 }
+
+/**
+ * Sets a given `value` at the given `path` on a passed `object`.
+ *
+ * @example
+ * ```js
+ * const obj = { a: { b: { c: 1 }}}
+ * setIn(obj, ['a', 'b', 'c'], 5)
+ * obj.a.b.c //> 5
+ * ```
+ *
+ * @template V
+ * @param {any} object
+ * @param {string[]} path
+ * @param {V} value
+ */
+export const setIn = (object, path, value) => {
```

Comment:
Can `client/test/service.js` use this?

Reply:
I was not sure how to go about without introducing client to server dependency. I'll just use package hugo linked

Reply:
oh, oops yes sorry ignore me.